third_party/2and3/requests/structures.pyi

```diff
@@ -1,6 +1,8 @@
 from typing import Any, Iterator, MutableMapping, Text, Tuple, Union
 
 class CaseInsensitiveDict(MutableMapping[str, Union[Text, bytes]]):
+    def __init__(self, data=Optional[
+        MutableMapping[Union[str, bytes], Any]]) -> None: ...
```

Comment:
Why did you make the key type `Union[str, bytes]`? It seems pretty clear that the other methods only accept keys for dicts. Similar for the value type -- why not `Union[Text, bytes]` like the rest of the API here?

Reply:
Also, you're making `Optional[...]` the default, value, not the type. You want something like `data: Optional[...] = ...`.

Why do you need a MutableMapping? I would expect it to accept any Mapping.